### PR TITLE
Update sidebar styles

### PR DESCRIPTION
**Pull request by negue**
Changed the sidebars (to use the components) on following pages:
- [x] Challenges
- [x] Groups
- [x] Shops/Market
- [x] Shops/Quests
- [x] Shops/Seasonal
- [x] Shops/TimeTravelers

### CODE REVIEW

**Comment by negue** [comment]
Those are done on the "style" part, code review left :)

File: website/client/src/components/shops/market/filter.vue [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-        :key="viewOptionKey"
-        :checked.sync="viewOptions[viewOptionKey].selected"
-        :text="viewOptions[viewOptionKey].text"
+  <filter-sidebar>
+    <filter-group>
+      <checkbox :checked.sync="viewOptions[viewOptionKey].selected"
+                :id="`category-${viewOptionKey}`"
+                :key="viewOptionKey"
+                :text="viewOptions[viewOptionKey].text"
+                v-for="viewOptionKey in Object.keys(viewOptions)"
```
Comment (paglias) [inline website/client/src/components/shops/market/filter.vue]: I know this isn't a change from this PR, but can we change it not to use `Object.keys`? v-for can be used to iterate over objects natively

**Review by paglias — COMMENTED** [review]
@negue left one comment which would be great if we could fix, but then it's ready!

File: website/client/src/components/shops/market/filter.vue (hunk 1 above)
Comment (negue) [reply website/client/src/components/shops/market/filter.vue]: yep will do

**Comment by paglias** [comment]
looks like there are some conflicts, then feel free to merge

**Comment by negue** [comment]
> looks like there are some conflicts, then feel free to merge

yeah haha I pushed the previous commit and then I saw the conflicts :D

**Pull request merged**